localstack/services/sns/sns_listener.py

```diff
@@ -226,6 +227,23 @@ def create_sns_message_body(subscriber, req_data):
     return json.dumps(data)
 
 
+def create_sqs_message_attributes(subscriber, attributes):
+    if subscriber['RawMessageDelivery'] == 'false':
```

Comment:
If the key `"RawMessageDelivery"` is missing in `subscriber`, this would raise a `KeyError`. Perhaps we can change it to:
```
if subscriber.get('RawMessageDelivery') not in ('true', True):
    ...
```